lib/ecto/schema.ex

```diff
@@ -195,6 +198,7 @@ defmodule Ecto.Schema do
       import Ecto.Schema, only: [schema: 2]
       @primary_key {:id, :integer, read_after_writes: true}
       @timestamps_type Ecto.DateTime
+      @timestamps_use_usec true
```

Comment:
> And you even documented the limitations in MySQL in the adapter. I could add a paragraph there saying something along the lines of: If you use MySQL 5.6 set usec: false on timestamps or make the types datetime(6) in your migrations.

That's the issue. Imagine I am writing some code that is meant to be used with any database because it is a library. I am not going to remember adding the usec: false and the code will be broken under certain databases (by broken I mean: if we care so much about this precision, having it half a second "too quick" sounds horrifying). This is also a huge incognito when talking about future adapters. One of the goals of adding MySQL was exactly so we get some diversity in and test our APIs with something besides Postgres. :)